### DISCUSSION TITLE
what does "[...].Home" in the destination field mean?

### DISCUSSION THREAD

**Discussion by Moth-Tolias** [evilsocket/opensnitch#1023] [Q&A] [answered]
see attached image.
`.home` isn't a valid tld as far as i'm aware, so i don't understand what is going on here
![image](https://github.com/evilsocket/opensnitch/assets/10255268/35769683-7bb1-4c13-a21c-fce53c780fb7)

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
Hi @Moth-Tolias ,

.home is a domain used for local networks, usually set by your router, for resolving hostnames to IPs. When connecting to the local network via DHCP, besides the IP/mask you also receive the DNS nameservers and a domain for your localnetwork. In your case is .home, but it could also be .local, or .homestation for example.

Nothing to worry about, probably you can configure it in your router. Also this domain may be added to your /etc/resolv.conf, with the parameter "search .home", to easily access other hosts in the local network by hostname.
